include/cantera/transport/HighPressureGasTransport.h

```diff
@@ -66,26 +258,1047 @@ class HighPressureGasTransport : public MultiTransport
+    * Calculates the high-pressure mixture thermal conductivity using the Chung method.
+    *
+    * This method obtains appropriate mixture values of the parameters needed for the
+    * Chung model and then calls the highPressureThermalConductivity() method to
+    * obtain the mixture thermal conductivity.
+    *
+    * The mixture values of the pseudo-critical temperature and other model parameters
+    * are calculated using the Chung mixing rules defined on page 9.25.
+    * @see computeMixtureParameters() .
+    *
+    * The mixture value of the specific heat is computed using equation 10-6.6, which
+    * is the mole fraction weighted sum of the pure species specific heats. This value
+    * is not directly computed by the computeMixtureParameters() method.
+    *
+    * @f[
+    *   C_{v,m} = \sum_i X_i C_{v,i}
+    *
+    *   \quad \text{(Equation 10-6.6)}
+    * @f]
+    *
+    * Where @f$ C_{v,i} @f$ is the specific heat of species i, and @f$ X_i @f$ is the
+    * mole fraction of species i.
+    *
+    */
+    double thermalConductivity() override;
+
+    /**
+     * Returns the matrix of binary diffusion coefficients, augmented by the
+     * Takahashi correction factor.
+     *
+     *      d[ld*j +  i] = rp*m_bdiff(i,j)*(DP)_R;
+     *
+     * @param ld    offset of rows in the storage
+     * @param d     output vector of diffusion coefficients.  Units of m**2 / s
+     */
+    void getBinaryDiffCoeffs(const size_t ld, double* const d) override;
+
     friend class TransportFactory;
 
 protected:
+
+    /**
+     * Obtain required parameters from the 'critical-parameters' species input section,
+     * and checks the critical-properties.yaml file if an acentric factor is not
+     * specified.
+     *
+     * The way that GasTransport parses the input file is that if an acentric
+     * factor is not specified, it is quietly set to 0.0. A user may have the proper
+     * acentric factor in the critical-properties.yaml file, so that file is checked if
+     * a zero value is present.
+     *
+     */
+    void getTransportData() override;
+
+    /**
+     * Computes and stores the estimate of the critical properties for each species.
+     *
+     * This method sets the species composition vector to unity for species i and zero
+     * for all other species, and then queries the thermo object for the critical
+     * properties and stores them. It then resets the composition vector to the original
+     * state. This method only needs to be called once, as the critical properties for
+     * the pure species do not change.
+     *
+     * All species must have critical properties defined in the input file, either via
+     * critical properties or by specific values of the equation of state that are
+     * not zero.
+     *
+     */
+    void initializeCriticalProperties();
+
+     /**
+     * Computes and stores pure-fluid specific properties each species.
+     *
+     *
+     */
+    void initializePureFluidProperties();
```

Comment:
This is a lot of extra vertical space for something that could be written as:
```suggestion
    //! Computes and stores pure-fluid specific properties each species.
    void initializePureFluidProperties();
```